pkgs/development/tools/abuild/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, pkg-config
+, openssl
+, zlib
+, busybox
+}:
+
+stdenv.mkDerivation rec {
+  pname = "abuild";
+  version = "3.7.0";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.alpinelinux.org";
+    owner = "alpine";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1xsik9hyzzq861bi922sb5r8c6r4wpnpxz5kd30i9f20vvfpp5jx";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    openssl
+    zlib
+  ];
+
+  patchPhase = ''
+    substituteInPlace ./Makefile \
+      --replace '-Werror' '-Wno-error' \
+      --replace 'chmod' 'echo'
+  '';
+
+  installPhase = ''
+    DESTDIR=$out make install
+  '';
+
+  postInstall = ''
+    wrapProgram $out/bin/abuild \
+      --prefix PATH : ${lib.makeBinPath [ busybox ]} \
+    substituteInPlace $out/bin/abuild \
+      --replace "/bin/ash" "${busybox}/bin/ash"
+    patchShebangs $out/bin/abuild
+  '';
+
+  meta = with lib; {
+    description = "Alpine Linux build tools";
+    homepage    = "https://gitlab.alpinelinux.org/alpine/abuild";
+    license     = licenses.gpl2Only;
+    maintainers = with maintainers; [ onny ];
+    platforms   = with platforms; unix;
```

not sure if `gpl2Only` is right here, upstream license has an openssl exception
```suggestion
    homepage = "https://gitlab.alpinelinux.org/alpine/abuild";
    license = licenses.gpl2Only;
    maintainers = with maintainers; [ onny ];
    platforms = platforms.unix;
```